src/api/namespace.ts

```diff
@@ -32,12 +32,16 @@ const updateNameSpace = (nameSpace, t) => {
         });
 };
 
-const getNameSpaces = () => {
+const getNameSpaces = (param?: string) => {
+    const params: Record<string, string> = {};
+
+    if (typeof param === "string" && param !== null) {
```

Comment:
nitpick: Could we make this verification outside the helper? Maintaining the business logic on the component .